examples/examples/discovery/README.md

```diff
@@ -0,0 +1,34 @@
+# Discovery
+
+## Running The Example
+
+This example demonstrates how to list all active services within your Elkodon
+application. To get the most out of this example, you can combine it with other
+examples, such as the [event example](../event/) or the
+[publish-subscribe example](../publish_subscribe/), to see active services in
+action.
+
+To begin, let's start some interesting services. Open two terminals and execute
+the following commands:
+
+**Terminal 1**
+
+```sh
+cargo run --example event_listener
+```
+
+**Terminal 2**
+
+```sh
+cargo run --example publish_subscribe_subscriber
+```
+
+Once these services are running, you can call the following command:
+
+```sh
+cargo run --example discovery
```

Comment:
It only shows the max number ob subscriber and publisher but not the current number of subscriber and publisher. I guess that is a known limitation?

Reply:
It is not a limitation but it is just a discovery. When you want to know these details you have to do introspection. Discovery is for me the just get to know what kind of services are present.

Reply:
right, I confused the discovery with the introspection